src/server_manager/web_app/digitalocean_server.ts

```diff
@@ -307,7 +307,7 @@ class DigitalOceanHost implements server.ManagedServerHost {
   getMonthlyOutboundTransferLimit(): server.DataAmount {
     // Details on the bandwidth limits can be found at
     // https://www.digitalocean.com/community/tutorials/digitalocean-bandwidth-billing-faq
-    return {terabytes: this.dropletInfo.size.transfer};
+    return {terabytes: this.dropletInfo.size.transfer * (2 ** 40)};
```

Comment:
We're converting to bytes. This should be `this.dropletInfo.size.transfer * (2**40 / 10**12)` to use terabytes. 
Maybe it would be clearer if we changed `server.DataAmount` to use `bytes` instead?

Reply:
This conversion should not be needed. The change to the formatting library is probably enough. Also, the conversion is wrong, since the size.transfer is already in terabytes (base 10), AFAIK.

Reply:
The docs say that the monthly allowance is 1,000 GiB. I we sholud update the model to use bytes and convert here.